policies/elections.tex

```diff
@@ -135,57 +148,57 @@ \subsubsection{Voting Period}
 During the voting period, no one shall actively campaign for the election of any specific candidate.
 Material posted or distributed prior to the voting period may remain posted during the voting period.
 At the conclusion of the voting period, it is a candidate's responsibility to clean up after their campaign.
+
 During the voting period, a candidate is allowed to answer questions that require a response that might otherwise be considered campaigning.
 
 \subsubsection{Encouragement to Vote}
 Encouraging eligible voters to vote, without supporting or denouncing a specific candidate or candidates, is not considered campaigning and may done freely by any party, including a candidate, throughout the entirety of the election, including the voting period.
+Likewise, any party, including a candidate, may inform eligible voters of any or all candidates that are running and it shall not considered campaigning, throughout the entirety of the election, including the voting period, as long as no specific candidate is supported or denounced.\footnote{For example, during the voting period a candidate can say ``I am running for Council, go to \votewebsite{} to vote!'', but cannot say ``I am running for Council, go to \votewebsite{} and vote for me!''.}
 
-The Elections Committee may, within the limits of its own spending authorizations, authorise the reimbursement of any party for any expenses incurred in the process of encouraging voters to vote.
+The Elections Committee may, within the limits of its own spending authorizations, authorize the reimbursement of any party for any expenses incurred in the process of encouraging voters to vote.
 
 \subsection{Voting}
 \subsubsection{Means of Voting}
-Voting shall be conducted by an electronic poll on either the Society’s website, or the Federation of Students’ website.
+\label{elections:meansOfVoting}
+Voting shall be conducted by an electronic poll on either the Society's or the Federation of Students' website.
 Voting shall be by means of preferential ballot, in which the voter ranks some or all of the candidates.
-If permitted by the voting software, a candidate may rank some candidates equally.
-The candidates shall be listed in random order for each voter.
-Each voter shall be permitted to vote in each Executive election, and in one Representative election.
-If a voter is in multiple constituencies, then the voter must select one in which to cast their vote.
-The resolution method shall be decided prior to the voting period and published on the voting website used.
-The vote shall be secret. 
+If permitted by the voting software and electoral system, a candidate may rank some candidates equally.
+If supported by the voting software, the candidates shall be listed in random order for each voter.
+In the event of significant technical issues, the Elections Committee may allow voting to be done on paper ballots.
```

Comment:
In the event of significant technical issues, the Elections Committee may allow voting to be done on paper ballots, or extend the voting period.